Dockerfile

```diff
@@ -9,14 +9,21 @@ FROM python:2.7
 
 # Install some prerequisites ahead of `setup.py` in order to profit
 # from the docker build cache:
-RUN pip install coveralls \
+RUN pip install Flask \
+                Flask-AppFactory \
```

Comment:
Indeed, it had better not... It was a remainder from `pip freeze` listing.  Fixed and re-pushed.